packages/terra-date-time-picker/tests/wdio/date-time-picker-spec.js

```diff
@@ -562,7 +607,8 @@ Terra.describeViewports('DateTimePicker', ['tiny', 'large'], () => {
     before(() => {
       browser.url('/#/raw/tests/terra-date-time-picker/date-time-picker/date-time-picker-default-date-out-of-range');
       browser.refresh();
-      browser.click('input[name="terra-date-input"]');
+      Terra.hideInputCaret('input[name="terra-date-month-input"]');
```

Comment:
This happens by default on refresh: https://github.com/cerner/terra-toolkit/blob/master/src/wdio/services/TerraService.js#L82

so this can be removed